docs/user/system_upgrades.md

```diff
@@ -0,0 +1,92 @@
+# Upgrading your system
+
+Brewblox is designed to be compatible with the majority of Linux-based systems.
+but there still are dependencies, and those have their own End of Life dates.
+
+When your system is (soon to be) no longer supported, we strongly recommend upgrading.
+In the long run, it takes much more effort to keep an old system running than it is to upgrade.
+
+To assist with upgrades, `brewblox-ctl` can save and load snapshots of your installation.
+This lets you easily move your configuration between systems.
+
+## Deprecation schedule
+
+We track two deprecation schedules: Debian (the base of the Raspberry Pi OS), and Python.
+Linux distros come with a built-in version of Python.
+For some systems, it's doable to upgrade Python versions.
+For the Raspberry Pi, it's significantly easier and faster to upgrade the entire OS.
+
+### Raspberry Pi OS
+
+The Raspberry Pi OS is based on Debian,
+and we support the releases based on the [stable Debian version](https://wiki.debian.org/DebianReleases).
+Debian versions become stable a year after release.
+This means their End of Life date is *one year after the release of the next version*.
+
+The End of Life dates for the last three Debian versions are:
+- Buster: ~2022
+- Stretch: 2020/07/06
+- Jessie: 2018/06/17
+
+### Python
+
+For Python, we support all [active Python3 versions](https://devguide.python.org/#status-of-python-branches).
+Python versions are active for five years after release.
+
+New versions may have some compatibility problems immediately after release.
+
+The End of Life dates for the last Python versions are:
+- 3.9: ~2025
+- 3.8: 2024/10
+- 3.7: 2023/06
+- 3.6: 2021/12
+- 3.5: 2020/09
+
+For reference, these are the default Python versions for various distros:
+
+- Debian Stretch: Python 3.5
+- Debian Buster: Python 3.7
+- Ubuntu 18.04: Python 3.6
+- Ubuntu 20.04: Python 3.8
+
+If you are using Synology, MacOS, or one of the many flavours of Linux,
+then the Python version is the most important indicator of compatibility.
+
+## Upgrading Brewblox
+
+You can seamlessly move your Brewblox installation from one OS to another.
+This is done by making a snapshot of your installation directory, and loading it on your new system.
+
+### Creating a Snapshot
+
+To make a snapshot, run this command in your home directory (the one containing your brewblox directory):
+
+```
+brewblox-ctl snapshot save
```

Comment:
add snapshot load (as alias of install from snapshot?

Reply:
`snapshot load exists`, but does only that. `brewblox-ctl install` fixes system dependencies, and then calls either `brewblox-ctl init`, or `brewblox-ctl snapshot load`.